main.js

```diff
@@ -0,0 +1,23 @@
+console.clear();
```

Comment:
i think you don´t need this, this was only for the codepen example to clear the console for a better view :)

Reply:
thank you :)